composer.json

```diff
@@ -21,7 +21,7 @@
         "psr/container": "^1.0",
         "psr/http-message": "^1.0.1",
         "zendframework/zend-authentication": "^2.5.0",
-        "zendframework/zend-expressive-authentication": "^0.2 || ^1.0.0-dev || ^1.0"
+        "zendframework/zend-expressive-authentication": "^0.3 || ^1.0.0-dev || ^1.0"
```

Comment:
Why not both? I mean `^0.2 || ^0.3 || ....` and the others.

Reply:
Because we changed the [UserInterface](https://github.com/zendframework/zend-expressive-authentication/blob/master/src/UserInterface.php) definition in `zend-expressive-authentication` ver. 0.3.

Reply:
Yes, but `UserInterface` is not used in this particular library (as you've updated only composer and no other changes) - so the repository supports 0.2 and 0.3

Reply:
Yes, but we want to be consistent with other authentication adapters and use the latest interface for future needs.